src/client/components/070-details.js

```diff
@@ -119,6 +123,13 @@ const style = {
   srcList: {
     marginTop: 0,
   },
+  description: {
+    marginBottom: 10,
+  },
+  descriptionTitle: {
+    fontWeight: 900,
+    marginRight: 10,
```

Comment:
I suggest adding `color: '#444'` (same color as for the dates) (remember to update the test snapshots)